tests/e2e/testutils.go

```diff
@@ -391,7 +400,18 @@ func ParsePods(kubeConfig string, print bool) ([]Pod, error) {
 // RestartCluster restarts the k3s service on each node given
 func RestartCluster(nodeNames []string) error {
```

Comment:
Should this be RestartServer, since it only works on server nodes?

Reply:
Thanks Brad, but I did not create the RestartCluster, if I change the name it'll require to change it in any other file that is using the func. I did add the docstring.